wallet/how-to/use-3rd-party-integrations/js-infura-api.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 2
+description: Use Infura and custom nodes to make direct, read-only requests.
+---
+
+# Make read-only requests from your JavaScript dapp
+
+You can use the [Infura API](https://docs.infura.io/) from your JavaScript dapp with
+[MetaMask SDK](../connect/set-up-sdk/javascript/index.md) installed to make direct, read-only
+JSON-RPC requests.
+
+Direct, read-only JSON-RPC requests are blockchain requests that do not require user wallet interaction.
+Your dapp can directly call most [JSON-RPC API methods](/wallet/reference/eth_subscribe), bypassing
+user wallet authentication for read-only operations.
+
+<details>
+<summary>JSON-RPC methods you <b>cannot</b> directly call</summary>
+<p>
+
+Your dapp can directly call all JSON-RPC API methods EXCEPT the following methods, which require
+user wallet interaction:
+
+- `eth_requestAccounts`
+- `eth_sendTransaction`
+- `eth_signTransaction`
+- `eth_sign`
+- `eth_accounts`
+- `personal_sign`
+- `eth_signTypedData_v4`
+- `wallet_watchAsset`
+- `wallet_addEthereumChain`
+- `wallet_switchEthereumChain`
+
+</p>
+</details>
+
+You can configure your dapp to make read-only requests using the [Infura API](#use-the-infura-api),
+[custom nodes](#use-custom-nodes), or [both](#use-the-infura-api-and-custom-nodes).
+
+## Prerequisites
+
+- An Infura API key.
+  Create one by following the first two steps in the
+  [Infura getting started guide](https://docs.infura.io/getting-started).
+- [MetaMask SDK set up](../connect/set-up-sdk/javascript/index.md) in your JavaScript dapp.
+
+## Use the Infura API
+
+To use the Infura API to make read-only requests, specify your Infura API key using the
```

Comment:
Just wondering, Infura has a security option that forces you to specify an additional API key secret for all requests. How do we handle this if that option is set?

See: https://docs.infura.io/networks/ethereum/how-to/secure-a-project/project-secret

Reply:
It is still unknown how to handle these cases, the SDK will talk to the Infura team to find out.